generator/generateTokenTypes.ts

```diff
@@ -164,6 +164,7 @@ const types = {
   _public: new KeywordTokenType("public"),
   _private: new KeywordTokenType("private"),
   _protected: new KeywordTokenType("protected"),
+  _override: new KeywordTokenType("override"),
```

Comment:
I was going to say that it's weird there to be both a regular keyword and a contextual keyword for `override`, and ideally it would be only a contextual keyword because it's not a true language keyword. But I guess other modifiers use the same strategy (including non-keywords like `readonly`), so seems very reasonable to be consistent with that. FWIW, it feels fragile because the transformer won't necessarily know when to use `this.tokens.matches1(tt._override)` and when to use `this.tokens.matchesContextual(ContextualKeyword._override)`, so maybe it's something that could be cleaned up in the future.